platforms/ios/src/TangramMap/TGSceneUpdateStatus.mm

```diff
@@ -0,0 +1,55 @@
+//
+//  TGSceneUpdateStatus.h
+//  TangramMap
+//
+//  Created by Karim Naaji on 3/07/17.
+//  Copyright (c) 2017 Mapzen. All rights reserved.
+//
+
+
+#import "TGSceneUpdateStatus.h"
+#import "TGSceneUpdate.h"
+#import "TGHelpers.h"
+
+@interface TGSceneUpdateStatus ()
+
+@property (assign, nonatomic) TGSceneUpdate* sceneUpdate;
+@property (assign, nonatomic) TGSceneUpdateError error;
+
+@end
+
+@implementation TGSceneUpdateStatus
+
+- (instancetype)initWithSceneUpdate:(TGSceneUpdate *)sceneUpdate sceneUpdateError:(TGSceneUpdateError)error
+{
+    self = [super init];
+
+    if (self) {
+        self.sceneUpdate = sceneUpdate;
+        self.error = error;
+    }
+
+    return self;
+}
+
++ (NSArray<TGSceneUpdateStatus *> *)sceneUpdateStatuses:(const std::vector<Tangram::SceneUpdateStatus>&)updateStatuses
+{
+    NSMutableArray* nsUpdateStatuses = [[NSMutableArray alloc] initWithCapacity:(NSUInteger)updateStatuses.size()];
```

Comment:
Nitpick - probably want to do a check here on `updateStatuses` to make sure it's not nil or otherwise this will crash. Objective-c will let you do the message send using something like `performSelector` so a little defensive programming here might be wise.